### PR TITLE
add flexible TwoColumn

### LINKED ISSUE #287: Custom draggable divider in two column
The divider in the two column layout could be draggable. By dragging the user could dynamically set the size.

- Still allow the fixed size
- ensure that the side bar is still visible on the edges

### PR DESCRIPTION
**Pull request by DasProffi**
closes #287

### CODE REVIEW

File: tasks/components/layout/TwoColumn.tsx [hunk 1]
```diff
@@ -21,49 +30,89 @@ type TwoColumnProps = {
...
+          // Overscrolling to the right makes the value 0, so we block this
+          if (event.pageX !== 0) {
+            setLeftWidth(calcPosition(event.pageX))
+          }
+          event.preventDefault()
+        }}
+        className={tx(`relative h-full flex justify-center bg-white w-[${dividerHitBoxWidth}px]`, { 'cursor-col-resize': !disableResize })}
+      >
+          <div className={tw('bg-gray-300 my-4 rounded-lg w-0.5')} />
+          <div className={tw('absolute top-[50%] bg-gray-300 rounded-xl w-6 h-20 -translate-y-[50%]')} />
```
Comment (MaxSchaefer) [inline tasks/components/layout/TwoColumn.tsx:100]: A little bit smaller
```suggestion
          <div className={tw('absolute top-[50%] bg-gray-300 rounded-xl w-3 h-12 -translate-y-[50%]')} />
```

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
- Drag pill should be invisible when `disableResize` is set to `true``
- Higher constraints defaults. Right now, ~`33vw` feels right.

File: tasks/components/layout/TwoColumn.tsx (hunk 1 above)
Comment (DasProffi) [reply tasks/components/layout/TwoColumn.tsx:100]: Making the draggable smaller would increase the effort required by the user to drag

Comment (DasProffi) [reply tasks/components/layout/TwoColumn.tsx:100]: Nevermind you were only concerned with the drag pill and not the entire draggable zone

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**